ui-v2/app/components/topology-metrics/series/index.js

```diff
@@ -147,20 +151,20 @@ export default Component.extend({
 
     let self = this;
     svg
-      .on("mouseover", function(){
-        tooltip.style("visibility", "visible");
-        cursor.style("visibility", "visible");
+      .on('mouseover', function() {
+        tooltip.style('visibility', 'visible');
+        cursor.style('visibility', 'visible');
         // We update here since we might redraw the graph with user's cursor
         // stationary over it. If that happens mouseover fires but not
         // mousemove but the tooltip and cursor are wrong (based on old data).
         self.updateTooltip(event, data, stackData, keys, x, tooltip, tipVals, cursor);
       })
-      .on("mousemove", function(d, i){
+      .on('mousemove', function(d, i) {
         self.updateTooltip(event, data, stackData, keys, x, tooltip, tipVals, cursor);
```

Comment:
@banks Re https://github.com/hashicorp/consul/pull/8886/files#r502369139, `updateTooltip` appears to get registered here on `mouseover` and `mousemove` events (which can fire quite rapidly, might need to be throttled) regardless of series data validity (had to comment out these two lines initially when I had everything broken).